blog/content/post/2020-10-13-xen-implementation-for-trenchboot.md

```diff
@@ -0,0 +1,123 @@
+---
+title: 'Trenchboot: Xen hypervisor support for the TrenchBoot'
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/trenchboot-logo.png
+author:
+    - marek.kasiewicz
+    - norbert.kaminski
+
+layout: post
+published: true
+date: 2020-10-13
+archives: "2020"
+
+tags:
+  - Trenchboot
+  - Xen
+  - firmware
+  - AMD64
+categories:
+  - Firmware
+  - Security
+
+---
+
+If you haven’t read previous blog posts from the TrenchBoot series, we strongly
+encourage you to catch up on it. The best way is to search under the TrenchBoot
+tag. In this blog post, we will describe the development of the Xen hypervisor
+support for TrenchBoot.
+
+## Global interrupt flag reinitialization
+
+As we have mentioned in the previous 
+[blog post](https://blog.3mdeb.com/2020/2020-09-07-trenchboot-multiboot2-support/),
+since now the LZ has re-enabled the interrupts during the multiboot. As
+indicated in the [documentation](https://www.amd.com/system/files/TechDocs/24593.pdf#G21.1088220)
+the global interrupts flag (GIF) is a bit that controls whether interrupts and
+other events can be taken by the processor. It is cleared during the SKINIT
+boot. When the GIF is not set again, the Xen hypervisor will not able to
+perform the self-NMI test:
```

Comment:
@Asiderr IMO there is not enough context, you should mention to the user in high-level explanation why you have to perform those actions. What is hard about that and what is your plan to implement that.

Reply:
I've added the more context for both paragraphs.